ray-operator/test/e2e/support.go

```diff
@@ -175,3 +178,22 @@ func jobSubmitterPodTemplateApplyConfiguration() *corev1ac.PodTemplateSpecApplyC
 						corev1.ResourceMemory: resource.MustParse("500Mi"),
 					}))))
 }
+
+func curlRayServicePod(
+	t Test,
+	rayService *rayv1.RayService,
+	curlPod *corev1.Pod,
+	curlPodContainerName, //nolint:unparam // This parameter is necessary
```

Comment:
Yes, I got the error when running the linter.
```
test/e2e/support.go:186:2: `curlRayServicePod` - `curlPodContainerName` always receives `curlContainerName` (`"curl-container"`) (unparam)
        curlPodContainerName,
```
Maybe this can be removed in the future when someone else calls this function with a different curlPodContainerName